.circleci/run_style_check.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+import argparse
+from git.repo import Repo
+import subprocess
+import os
+import sys
+
+
+def run_black_formatter(apply_diff):
+    """
+    Runs black on committed files for style checks.
+    """
+    # Populate file list from the latest commit.
+    stream = os.popen(
+        "git diff-tree --no-commit-id --name-only --diff-filter=ACM -r HEAD"
+    )
+    files = [
+        y
+        for y in (x.strip() for x in stream.read().splitlines())
```

Comment:
```suggestion
        for y in (x.strip() for x in stream.readlines())
```

Reply:
Also, did you really find that the `strip()` was necessary?  If `git diff-tree --name-only` is documented to print one filename per line, there should be need for further manipulation beyond the splitlines.

Reply:
`strip()` should be unnecessary here. Removed it.